solutions/ids/endpoint/endpoint.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+# Cloud IDS endpoint
+# Warning! requires the alpha resource loaded in the config controller
+# https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/crds/cloudids_v1alpha1_cloudidsendpoint.yaml
+apiVersion: cloudids.cnrm.cloud.google.com/v1alpha1
+kind: CloudIDSEndpoint
+metadata:
+  name: host-project-id-endpoint-name-ids # kpt-set: ${host-project-id}-${endpoint-name}-ids
```

Comment:
since `host-project-id` is also in the parent package setters, `${host-project-id}-${endpoint-name}-ids` may need to be changed to something like `${host-project-id}--${endpoint-name}-ids` to work around a known behavior described [here](https://github.com/GoogleContainerTools/kpt/issues/3330)
OR
add a comment in setters.yaml to prohibit a dash in the `endpoint-name` value.

Reply:
Could we potentially add some local validation to check for that?

Reply:
added "--" as recommended. @cartyc I cannot think of simple way to validate that ?

Reply:
I was thinking something probably broader and using either OPA or a `starlark` validator. Doesn't have to be part of this PR.

Reply:
ok, marking this conversation as resolved for now